dpctl-capi/helper/include/dpctl_utils_helper.h

```diff
@@ -138,3 +138,30 @@ sycl::aspect DPCTL_DPCTLAspectTypeToSyclAspect(DPCTLSyclAspectType AspectTy);
  * @throws runtime_error
  */
 DPCTLSyclAspectType DPCTL_SyclAspectToDPCTLAspectType(sycl::aspect Aspect);
+
+/*!
+ * @brief Converts a DPCTLPartitionAffinityDomainType enum value to its
+ * corresponding sycl::info::partition_affinity_domain enum value.
+ *
+ * @param    PartitionAffinityDomainTy           A
+ * DPCTLPartitionAffinityDomainType enum value
+ * @return   A sycl::info::partition_affinity_domain enum value for the input
+ * DPCTLPartitionAffinityDomainType enum value.
+ * @throws runtime_error
+ */
+sycl::info::partition_affinity_domain
```

Comment:
Please add the `DPCTL_API` attribute. I committed a change for the other functions in this file to master. Without the `DPCTL_API` attribute we cannot run test cases on Windows.